test/unit/nedap/utils/test/api.cljc

```diff
@@ -202,10 +218,13 @@
                "#{:to-change :from :to}"
                `(sut/expect () :unexpected () :signature 4 :keys)
 
-               "0 should be different from 0"
+               "0 should not match 0"
```

Comment:
Looks like the message should also be part of the multimethod, since the idea of matching is extraneous to `=` / `meta=`

Reply:
I don't think that's necessary yet; but we can expand `expect-matcher` to this in the future for sure!

Reply:
I like the idea of having a customizable message. However, I agree on putting that on a future issue.